pkg/utils/decode/bytes.go

```diff
@@ -0,0 +1,39 @@
+package decode
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+)
+
+// ParseBytes20 parses `string` into `[20]byte` type. The input string must have
+// 20 bytes hex-encoded.
+func ParseBytes20(str string) ([20]byte, error) {
+	bytesArray := [20]byte{}
+	slice, err := hexutil.Decode(str)
```

Comment:
Why not just `hex.DecodeString` from the standard library? Is there any particular reason to use `hexutil.Decode` here?

Reply:
`hexutil` package is already used to [decode `[]byte`](https://github.com/keep-network/keep-common/blob/2632871d696cd604823591142e2e1f7b61c96b60/tools/generators/ethereum/contract_parsing.go#L233) and to [decode `*big.Int`](https://github.com/keep-network/keep-common/blob/2632871d696cd604823591142e2e1f7b61c96b60/tools/generators/ethereum/contract_parsing.go#L237). It also works with `0x` prefix while `hex.DecodeString` from the standard library expects the prefix to be skipped.

I was thinking about using the standard library version here but then I'd have to deal with prefix, and clean up `contract_parsing.go` by introducing another function in `bytes.go` that would deal with `[]byte` for consistency. Then, I'd still have the `*big.Int` parsing left.

I know it is not ideal but it's simpler. And I think we are bound to `hexutil` anyway so we should either use it consistently or replace other usages as well.

Reply:
Ok, that's clear now. Thanks!